src/main/java/uk/gov/companieshouse/api/accounts/service/impl/CurrentPeriodServiceImpl.java

```diff
@@ -32,6 +32,11 @@ public void addKind(CurrentPeriod rest) {
         rest.setKind(Kind.CURRENT_PERIOD.getValue());
     }
 
+    @Override
+    public void addID(CurrentPeriodEntity entity) {
+
```

Comment:
We shouldn't be overriding with empty methods. Are these going to have any functionality?

Reply:
fixed

Reply:
I'm not sure why these changes are coming up as I have not edited the files at all, however,I have removed the method